hack/scripts/releaser/releaser.go

```diff
@@ -806,22 +369,14 @@ func run() error {
 	flag.Var(release.targetVersion, "target-version", "Release target version")
 	flag.Var(release.currentVersion, "current-version", "Current version")
 	flag.Var(release.helmChartsTargetVersion, "helm-charts-release-version", "Helm charts release target version")
-	flag.StringVar(&release.githubToken, "github-token", common.GetEnvOrDefaultString("NUCLIO_RELEASER_GITHUB_TOKEN", ""), "A scope-less Github token header to avoid API rate limit")
-	flag.StringVar(&release.repositoryOwnerName, "repository-owner-name", "nuclio", "Repository owner name to clone nuclio from (Default: nuclio)")
-	flag.StringVar(&release.repositoryScheme, "repository-scheme", "git", "Scheme to use when cloning nuclio repository")
-	flag.StringVar(&release.developmentBranch, "development-branch", "development", "Development branch (e.g.: development, 1.3.x")
-	flag.StringVar(&release.releaseBranch, "release-branch", "master", "Release branch (e.g.: master, 1.3.x, ...)")
-	flag.BoolVar(&release.skipCreateRelease, "skip-create-release", false, "Skip build & release flow (useful when publishing helm charts only)")
+	flag.StringVar(&release.releaseInfoPath, "release-info-path", "", "Path to save release info")
```

Comment:
why not have a default path?

Reply:
@TomerShor if we don't pass it, file won't be populated